portfolio/src/main/webapp/style.css

```diff
@@ -188,3 +188,9 @@ label {
   padding: 8px;
   font-family: Arial;
 }
+
+#map {
+  border: thin solid black;
+  height: 500px;
+  width: 500px;
+}
```

Comment:
tiny nit, this file should end in a new line

Reply:
addressed in "formatting"